src/modules/content-tree/components/content-tree/content.tree.js

```diff
@@ -61,6 +61,19 @@ export default class ContentTree extends Component {
         window.document.body.classList.remove(CLASS_IS_TREE_RESIZING);
     }
 
+    renderNoSubitemMessage() {
+        const { items } = this.props;
+        const rootLocation = items[0];
+        const itemsLoaded = !!rootLocation;
```

Comment:
As I see you need a negation of `itemsLoaded` so maybe just `const itemsNotLoaded = !rootLocation`?
But now I'm wondering why you want to return if items are not loaded and checking the root? Probably the name should be `rootNotLoaded`?

Reply:
Name was confusing, I changed it and deleted double `!!` because they are not necessary in this case. 
I didn't opt for using `not` negation in variable name because I find it hard to process in my mind when analysing code.